src/Accounts/Accounts/Models/RMProfileClient.cs

```diff
@@ -168,38 +171,44 @@ public bool TryRemoveContext(IAzureContext context)
                 // (tenant is present and subscription is not provided)
                 if (!string.IsNullOrEmpty(tenantIdOrName))
                 {
-                    Guid tempGuid = Guid.Empty;
-                    if (!Guid.TryParse(tenantIdOrName, out tempGuid))
+                    IAccessToken token = null;
+                    try
                     {
-                        var tenants = ListAccountTenants(account, environment, password, promptBehavior, promptAction);
-                        var matchesName = tenants.Where(t => t.GetPropertyAsArray(AzureTenant.Property.Domains)
-                                                                                           .Contains(tenantIdOrName, StringComparer.InvariantCultureIgnoreCase));
-                        var homeTenants = matchesName.FirstOrDefault(t => t.IsHome);
-                        var tenant = homeTenants ?? matchesName.FirstOrDefault();
-                        if (tenant == null || tenant.Id == null)
+                        token = AcquireAccessToken(
```

Comment:
If user input a domain name, will AcquireAccessToken() still be successful?

Reply:
> If user input a domain name, will AcquireAccessToken() still be successful?

yes